### PR TITLE
Text file on block structure

### PR DESCRIPTION
**Pull request by josojo**

### CODE REVIEW

File: blockStructures.md [hunk 1]
```diff
@@ -49,6 +49,6 @@ AuctionResultBlock (with AuctionOutputBlock):
 
 - root = sha(sha(MerkleRoot(prices),MerkleRoot(Volumes)), MerkleRoot(auctionOutputBlock))
 - auctionOutputBlocks contains all outputs created by an auction. This will be a tree of depth 17
```
Comment (fleupold) [inline blockStructures.md]: I'd add a small explanation why depth is 17.

File: blockStructures.md [hunk 2]
```diff
@@ -5,34 +5,34 @@ Block structure overview:
...
--orders are listed 0-#(Orders), rest of merkle tree is filled with 0's
+- these two MerkleRoots are made by a Merkle tree of height 16
+- orders are listed 0-#(Orders), rest of merkle tree is filled with 0's
+
 
 Each order is made of two parts: skeletonOrder, generalOrder
 
 
-skeletonOrder 256 bits needed for snark proofs:
+*skeletonOrder* 256 bits needed for snark proofs:
```
Comment (fleupold) [inline blockStructures.md:21]: 253 bits

File: blockStructures.md [hunk 3]
```diff
@@ -5,34 +5,34 @@ Block structure overview:
...
+*generalInfo*:
 
-owner (integer id coming from the smart contract struct)
-UTXO (unique identifier of the referred order)
-signature owner,
-double sig of previous owner
+- owner (integer id coming from the smart contract struct)
+- UTXO (unique identifier of the referred order)
+- signature owner,
+- double sig of previous owner
```
Comment (fleupold) [inline blockStructures.md]: I'd add a comment saying what happens in case of UTXO that come directly from a previous auction

**Comment by bh2smith** [comment]
@josojo Could we put this kind of stuff in the WIki?

**Comment by fleupold** [comment]
Probably no longer relevant

**Pull request closed without merging**